app/components/Article/KeepGoing/keepGoing.module.css

```diff
@@ -4,3 +4,14 @@
   padding: var(--spacing-32);
   margin-bottom: var(--spacing-56);
 }
+
+@media (max-width: 640px) {
```

Comment:
Why the use of `@media only screen` in this other PR: https://github.com/StampyAI/stampy-ui/pull/451, but not in this one?

Reply:
Honestly no idea.. my IDE (intellij idea) decided to mark it as an error and block the commit unless I put `only screen`